Makefile

```diff
@@ -34,8 +34,11 @@ $(BIN_DIR):
 
 install: gfak
 	mkdir -p $(DESTDIR)$(PREFIX)/include
+  mkdir -p $(DESTDIR)$(PREFIX)/bin
```

Comment:
This line is indented with two spaces rather than a tab character, which won't work with `make`.